app/src/main/java/ch/epfl/skysync/screens/crewpilot/Chat.kt

```diff
@@ -34,7 +34,8 @@ fun ChatScreen(navController: NavHostController, viewModel: ChatViewModel) {
             onClick = { selectedGroup ->
               navController.navigate(Route.CREW_TEXT + "/${selectedGroup.id}")
             },
-            paddingValues = padding)
+            paddingValues = padding,
+            false)

```

Comment:
always more readable to put the parameter name i.e 
`adminBoolean = false` or alternatively define `val isAdmin = false `and use `isAdmin` instead of just `false`